files/zh-cn/web/css/list-style-type/index.md

```diff
@@ -67,88 +65,143 @@ list-style-type 的属性值可以是以下任意一项：
   - : 实心方块
 - `decimal`
   - : 十进制阿拉伯数字
+    - 从 1 开始
 - `cjk-decimal` {{experimental_inline}}
   - : 中日韩十进制数
+    - 例如：一，二，三，..., 九八，九九，一〇〇
 - `decimal-leading-zero`
-  - : Decimal numbers. Padded by initial zeros
+  - : Decimal numbers
+    - Padded by initial zeros
+    - E.g. 01, 02, 03, … 98, 99
 - `lower-roman`
   - : Lowercase roman numerals
+    - E.g. i, ii, iii, iv, v…
 - `upper-roman`
   - : Uppercase roman numerals
+    - E.g. I, II, III, IV, V…
 - `lower-greek`
   - : Lowercase classical Greek
-- `lower-alpha`、`lower-latin`
-  - : Lowercase ASCII letters. `lower-latin` is unsupported in IE7 and earlier
-
-    See [Browser compatibility](#browser_compatibility) section.
-
-- `upper-alpha`、`upper-latin`
-  - : Uppercase ASCII letters. `upper-latin` is unsupported in IE7 and earlier
+    - alpha, beta, gamma…
+    - E.g. α, β, γ…
+- `lower-alpha`, `lower-latin`
+  - : Lowercase ASCII letters
+    - E.g. a, b, c, … z
+    - `lower-latin` is unsupported in IE7 and earlier
+    - See [Browser compatibility](#browser_compatibility) section.
+- `upper-alpha`, `upper-latin`
+  - : Uppercase ASCII letters
+    - E.g. A, B, C, … Z
+    - `upper-latin` is unsupported in IE7 and earlier
 - `armenian`
   - : Traditional Armenian numbering
+    - (ayb/ayp, ben/pen, gim/keem…
 - `georgian`
   - : Traditional Georgian numbering
+    - E.g. an, ban, gan, … he, tan, in…
 - `hebrew` {{experimental_inline}}
   - : Traditional Hebrew numbering
 - `ethiopic-numeric` {{experimental_inline}}
-  - : Ethiopic numbering.
+  - : Example
 - `hiragana` {{experimental_inline}}
-  - : (Japanese)
+  - : a, i, u, e, o, ka, ki, …
+    - (Japanese)
 - `katakana` {{experimental_inline}}
-  - : (Japanese)
+  - : A, I, U, E, O, KA, KI, …
+    - (Japanese)
 - `hiragana-iroha` {{experimental_inline}}
-  - : [Iroha](https://zh.wikipedia.org/wiki/Iroha) is the old japanese ordering of syllabs.
+  - : i, ro, ha, ni, ho, he, to, …
+    - [Iroha](https://zh.wikipedia.org/wiki/Iroha) is the old japanese ordering of syllabs.
 - `katakana-iroha` {{experimental_inline}}
-  - : [Iroha](https://zh.wikipedia.org/wiki/Iroha) is the old japanese ordering of syllabs.
+  - : I, RO, HA, NI, HO, HE, TO, …
+    - [Iroha](https://zh.wikipedia.org/wiki/Iroha) is the old japanese ordering of syllabs.
 - `japanese-informal` {{experimental_inline}}
   - : Japanese informal numbering
 - `japanese-formal` {{experimental_inline}}
-  - : Japanese formal numbering to be used in legal or financial document.The kanjis are designed so that they can't be modified to look like another correct one
+  - : Japanese formal numbering to be used in legal or financial document.
+    - E.g., 壱萬壱阡壱百壱拾壱
+    - The kanjis are designed so that they can't be modified to look like another correct one
```

Comment:
```suggestion
  - : Japanese formal numbering to be used in legal or financial document. The kanjis are designed so that they can't be modified to look like another correct one
```